src/detection/DepthImagePolarHistDetector.cc

```diff
@@ -101,7 +101,17 @@ const std::vector<Obstacle> &DepthImagePolarHistDetector::detect(
         double phi = max_phi - (i * fixed_step) - (fixed_step / 2);
 
         Obstacle obs;
+
+        obs.bounding_box.tlc.x = histogram[i];
+        obs.bounding_box.tlc.y = (M_PI - depth_data->vfov) / 2 ;
```

Comment:
This seems to be swapped. If if the center is at M_PI / 2, left of the center should be bigger than that, not smaller. Also, if you add vfov/2 to both sizes, isn't the obstacle covering the whole sensor view? seems a bit too big.